core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -125,9 +125,9 @@ protected function __construct()
 	 *
 	 * @param string $strKey The property name
 	 *
-	 * @return mixed|null The property value or null
+	 * @return mixed The property value or null
 	 */
-	public function __get($strKey)
+	public function __get(string $strKey): mixed
```

Comment:
This breaks, as some subclasses like `User` are incompatible with their method signature.